contracts/interfaces/ILensHub.sol

```diff
@@ -523,4 +523,19 @@ interface ILensHub {
      * @return PubType The publication type, as a member of an enum (either "post," "comment" or "mirror").
      */
     function getPubType(uint256 profileId, uint256 pubId) external view returns (DataTypes.PubType);
+
+    /**
+     * @notice Returns the follow NFT implementation address.
+     *
+     * @return address The follow NFT implementation address.
+     */
+    function getFollowNFTImpl() external view returns (address);
+
+    /**
+     * @notice Returns the collect NFT implementation address.
+     *
+     * @return address The collect NFT implementation address.
+     */
+    function getCollectNFTImpl() external view returns (address);
```

Comment:
In case my last comment was correct, we can get rid of this one

Reply:
I think we keep this just for consistency, it doesn't make sense to me to only have the `getFollowNFTImpl()` function but not this.

Reply:
Ok, fair enough, I'm not against it